packages/hooking/hook.ts

```diff
@@ -93,3 +95,28 @@ export class Hook {
 		return pkg.includes(this.pkg) && target == this.target;
 	}
 }
+
+/**
```

Comment:
This function seems somewhat out of place. Move it to `manager` beside `registerBeforeHook`?

Reply:
done